.github/workflows/tests.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: pip
           cache-dependency-path: requirements-tests.txt
-      - run: pip install mypy packaging pathspec termcolor tomli
+      - run: pip install $(grep mypy== requirements-tests.txt) packaging pathspec termcolor tomli typing-extensions
```

Comment:
Flake8 is the problem right?
Can't you add a python version environment marker to the requirements file?

`flake8>=6 ; python_version>="3.8"

Reply:
Good point! Wanna make a PR?